csrc/python_frontend/python_bindings.cpp

```diff
@@ -3001,6 +3002,94 @@ void initNvFuserPythonBindings(PyObject* module) {
       },
       py::arg("tensor"),
       py::arg("memory_type"));
+  nvf_sched.def(
+      "transform_like",
+      [](FusionDefinition::SchedOperators& self,
+         Tensor tensor,
+         const std::vector<Tensor>& subgraph_nodes) {
```

Comment:
nitpick: the name of second arg is a bit confusing, it's just `selected_tensors`.

Also, the signature in PR description is wrong.
`transform_like(a: Tensor, subgraph_nodes : None | [Tensor])`. We do NOT have optional on second arg.

Reply:
> transform_all_like(a: Tensor) --- Transform all tensors in fusion like reference
transform_subset_like(a: Tensor, subset_nodes: [Tensor]) --- Transform a subset of tensors like reference
transform_all_except_like(a: Tensor, exception_subset: [Tensor]) --- Transform all tensor except those in subset like reference.

Should we just change the signature as
`transform_like(a: Tensor, selected_tensors: [Tensor] = (), excluded_tensors: [Tensor] = ())`. Where `selected_tensors.empty()` would just mean it's all tensors are targeted.

i.e. let's try to keep the APIs somewhat consistent with each other.

Reply:
I didn't have a need for `transform_all_except_like` with this set of PRs, so I left it alone for now.

Reply:
more nitpick here. Looks like we updated the description in PR as
`transform_like(a: Tensor, selected_nodes: [Tensor] = [])`

But the signature here is
`transform_like(tensor: Tensor, subgraph_nodes: [Tensor] = [])`